ansible/roles/contiv/tasks/netplugin.yml

```diff
@@ -0,0 +1,81 @@
+- name: ensure pluginDir exists
+  file: path=/usr/libexec/kubernetes/kubelet-plugins/net/exec recurse=yes state=directory
```

Comment:
Sure, I can do that. However, if I remember right, this path is hard-coded in kubelet.